authenticator/views.py

```diff
@@ -1,46 +1,68 @@
 #Django
 from django.shortcuts import render
 from django.contrib.auth import authenticate, login
-from django.contrib.auth.models import User
+from django.contrib.auth.models import User, AnonymousUser
 from django.shortcuts import render
 
 #Local model
 from authenticator.models import Personal_data
 
+from django.db.utils import IntegrityError
+
+
 
 #User creation in admin 
-def user_creation(request):
+def login_view(request):
 
     if request.method == 'POST':
         
         username = request.POST['Username']
         password = request.POST['password']
         user = authenticate(username=username, password=password)
-        if user:
+        if user is not None:
             login(request, user)
-            return render(request, 'login.html')
+            return render(request, 'login.html', {'error': 'Existing User'})
```

Comment:
If user exists, and then you login, why would you return an error that the user already exists? Isn't the login view just for login and not for user creation?